app/src/main/res/menu/navigation_drawer_menu.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:tools="http://schemas.android.com/tools"
+  tools:showIn="navigation_view">
+
+  <group android:checkableBehavior="single">
+
+    <item
+      android:id="@+id/nav_home"
+      android:icon="@drawable/ic_home_grey_48dp"
+      android:title="@string/menu_home" />
+    <item
```

Comment:
Please remove all items except home & help. We can add the others later. Ditto for their corresponding icons.

Reply:
resolved